frame/babe/src/mock.rs

```diff
@@ -230,6 +230,8 @@ parameter_types! {
 	pub const ExpectedBlockTime: u64 = 1;
 	pub const ReportLongevity: u64 =
 		BondingDuration::get() as u64 * SessionsPerEra::get() as u64 * EpochDuration::get();
+	pub const MaxAuthorities: u32 = 10;
+	pub const MaxSegmentLength: u32 = 256;
```

Comment:
this MaxSegmentLength should have been removed too

Reply:
Indeed, I have amended that in a PR #9850, thanks